Makefile

```diff
@@ -48,11 +48,17 @@ CATALOG_VERSION?=$(CHANNEL)-$(BUILD_DATE)-$(CURRENT_COMMIT)
 ALLOW_DIRTY_CHECKOUT?=false
 SOURCE_DIR=operators
 
+# List of github.org repositories containing operators
+# This is in the format username/reponame
+OPERATORS := openshift/dedicated-admin-operator
+
 .PHONY: default
 default: build
 
 .PHONY: clean
 clean:
+	ifndef SOURCE_DIR
+	$(error SOURCE_DIR needs to be defined or you delete /)
```

Comment:
Move this out side of the target and put with the other `ifndef` bits?  It is used in many places, makes sense it has to always be defined.

Reply:
@lisa looking at this again and it's something we could remove by having `SOURCE_DIR:=operators`, immediate set, instead of current lazy set.